striker-ui/Makefile.am

```diff
@@ -1,136 +1,66 @@
 MAINTAINERCLEANFILES	= Makefile.in
 
-# The files listed under EXTRA_DIST is only used for building, they should not
-# be included in the installation process.
-#
-# It is recommended to use the following command to regenerate the file list
-# as files get added/removed:
-#
-# find . \
-#	-not \( -name node_modules -prune \) \	# Ignore ./node_modules directory
-#	-not \( -name *git* -prune \) \		# Ignore git related files
-#	-not \( -name *husky* -prune \) \	# Ignore husky related files
-#	-not \( -name public -prune \) \	# Ignore ./public directory
-#	-not \( -name *[Mm]ake* -prune \) \	# Ignore make related files
-#	-not \( -name *.md -prune \) \		# Ignore all markdown files
-#	-type f \				# Only print file paths
-#	| sed -E 's@^./(.+)$@\1 \\@'		# Remove leading "./" and append "\"
+htmldir			= $(localstatedir)/www/html
+nodemodulesdir		= node_modules
+nextoutdir		= out
+nextbuilddir		= .next
+
+# List of paths relative to the build output directory.
 #
-EXTRA_DIST		= \
```

Comment:
I am not sure why you dropped those files from EXTRA_DIST, you still want them as part of the release tarball (source code) even tho they are not used directly.

Reply:
All files re-added.

The difference against the previous approach is directories are included as a whole instead of listing out all files within them. According to https://www.gnu.org/software/automake/manual/html_node/Basics-of-Distribution.html#index-EXTRA_005fDIST, copying the whole of each sub-directory is the desired bahaviour in this case.

Reply:
ahh.. this I missed.. good to know. Up to you how you prefer to do it.